src/gamefile/file.game

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>
+<game
+  type="unvanquished"
+  index="1"
+  name="Unvanquished"
+  enginepath_win32="C:/Program Files/Unvanquished/"
+  enginepath_linux="/usr/lib/unvanquished/"
+  enginepath_macos="/Applications/Unvanquished/"
+  engine_win32="daemon.exe"
+  engine_linux="../../../usr/bin/unvanquished"
+  engine_macos="daemon.app"
+  prefix=".unvanquished"
+  basegame="pkg"
+  basegamename="Unvanquished"
+  unknowngamename="Unvanquished Mod"
+  shaderpath="scripts"
+  archivetypes="pk3"
+  texturetypes="tga jpg png pcx bmp webp"
```

Comment:
If advertized `webp` support is missing in NetRadiant, NetRadiant will crash. Perhaps would it be safer to not advertise this format yet, or makes NetRadiant just raise a warning on unknown formats.